examples/basic/sdk/ontology.json

```diff
@@ -156,6 +156,11 @@
               "type": "string"
             }
           },
+          "name": {
+            "dataType": {
+              "type": "string"
+            }
+          },
```

Comment:
is this logically a duplicate of `usState`?

Reply:
I guess yea logically, usState is the two letter abbreviation though and I wanted to make sure it worked with strings that had spaces in it. I can remove though